Arduino/libraries/MPU6050/MPU6050.h

```diff
@@ -0,0 +1,1046 @@
+// I2Cdev library collection - MPU6050 I2C device class
```

Comment:
Probably could get rid of all these MPU6050 Files

Reply:
Saving until we're sure this new IMU works.